roles/confluent.kafka_broker/defaults/main.yml

```diff
@@ -52,3 +52,7 @@ kafka_broker_sysctl_file: /etc/sysctl.conf
 kafka_broker_health_check_delay: 20
 
 kafka_broker_secrets_protection_file: /var/ssl/private/kafka-broker-security.properties
+
+### Group names to use when gathering dynamic groups
+kafka_broker_controller_group: kafka_broker_controller
+kafka_broker_non_controller_group: kafka_broker_non_controller
```

Comment:
```suggestion
### Ansible group name to put kafka controller host into, used when gathering dynamic groups
kafka_broker_controller_group: kafka_broker_controller

### Ansible group name to put non-controller kafka hosts into, used when gathering dynamic groups
kafka_broker_non_controller_group: kafka_broker_non_controller
```

we now have a python script that turns annotations into documentation, you call it with
```
python doc.py
```

Reply:
Cool, I didn't know. I will fix the corresponding variables in confluent.zookeeper role defaults.